homeassistant/components/media_player/mpd.py

```diff
@@ -306,7 +306,7 @@ def play_media(self, media_type, media_id, **kwargs):
                 self._currentplaylist = media_id
             else:
                 self._currentplaylist = None
-                _LOGGER.warning(str.format("Unknown playlist name %s.",
+                _LOGGER.warning(str.format("Unknown playlist name {0}.",
```

Comment:
It's better to remove the string format function and let the logger function handle the format operator as is standard.

Reply:
I wondered about that, but deferred to the original author on it. I'll update the PR.